README.md

```diff
@@ -11,6 +11,12 @@ The image uses the following software:
 - [horizon](https://github.com/stellar/go/tree/master/services/horizon)
 - Supervisord is used from managing the processes of the services above
 
+In order to get started quickly, you can deploy the docker image to a DigitalOcean server by clicking the button below. It will create a container in *ephemeral mode* on the *standalone network* that can be connected to. Note: you will need to create a DigitalOcean account if you don't have one.
+
+[![Deploy to DO](https://www.deploytodo.com/do-btn-blue.svg)](https://cloud.digitalocean.com/apps/new?repo=https://github.com/stellar/docker-stellar-core-horizon/tree/acharb-quickdeploy)
```

Comment:
💡 It may be worth moving this down into the Standalone section, so it is really clear this deploys a standalone image. It may be counterintuitive that the deploy button doesn't deploy the default configuration which is a pubnet validator.

❔ On that note, the main use case we have immediately is to make it easy to deploy a standalone network for private testing. Is it easy to have multiple buttons that refer to different deployments? If that is trivial we could have a button in each section below, one for pubnet, one for testnet, one for standalone. That would probably make the fact there is a button more intuitive to what is being deployed.

Reply:
> Is it easy to have multiple buttons that refer to different deployments? If that is trivial we could have a button in each section below, one for pubnet, one for testnet, one for standalone

to my knowledge the way it's set up now it wouldn't be possible. Since the `deploy.template.yaml` has the docker command, which gives the flag of which network to join, and there's only one yaml file per repo.

It could be possible with some refactoring 🤔 . Eg. the `deploy.template.yaml` can have multiple services that could help. Or we could have a branch per network, but could be annoying to maintain.

If okay I think that work/thinking could be done after this PR if we decide the prod and testnet buttons are wanted?